birdhouse/deployment/trigger-deploy-notebook

```diff
@@ -83,7 +77,7 @@ docker run --rm \
   -u root \
   -v $TMP_SCRIPT:/deploy-notebook:ro \
   -v $TMPDIR/$TUTORIAL_NOTEBOOKS_DIR:/$TUTORIAL_NOTEBOOKS_DIR:ro \
-  -v $DOCKER_JUPYTERHUB_USER_PERSISTENCE_VOLUME:$NOTEBOOK_DIR_MNT:rw \
+  -v "${JUPYTERHUB_USER_DATA_DIR}":$NOTEBOOK_DIR_MNT:rw \
```

Comment:
From memory, this script do not read `env.local` so do not know what is `JUPYTERHUB_USER_DATA_DIR`.

Reply:
I realized that... not sure what we should do here? This script doesn't have any knowledge of the pavics repo... previously it was using the docker volume so it was accessible globally.

Maybe the pavics repo path should be passed when running `install-deploy-notebook`? And check the current folder when this variable is not given?

Reply:
Would appending the variable in /env/environment be a solution?

Reply:
Please check https://github.com/bird-house/birdhouse-deploy/pull/16/commits/a34b2af59a19f233eb142cf5f58f6c1503ebdcdc, I used envsubst when installing the cron script in /etc/hourly

Reply:
Nice, very clever user of envsubst.